components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -288,12 +300,15 @@ export namespace GitpodServer {
         limit?: number;
         searchString?: string;
         pinnedOnly?: boolean;
+        projectId?: string;
     }
     export interface GetAccountStatementOptions {
         date?: string;
     }
     export interface CreateWorkspaceOptions {
         contextUrl: string;
+        branch?: string;
+        projectId?: string;
```

Comment:
**Nit**: As before, I still think these parameters do not make sense in `CreateWorkspaceOptions`, and also they aren't used anywhere in this PR. Please remove them for now. 😇

```suggestion
```

Reply:
will do...